RiotSharp/RiotApi.cs

```diff
@@ -68,7 +69,7 @@ public class RiotApi : IRiotApi
         private const int MaxNrMasteryPages = 40;
         private const int MaxNrRunePages = 40;
         private const int MaxNrLeagues = 10;
-        private const int MaxNrEntireLeagues = 10;
+        private const int MaxNrLeaguePositions = 10;
```

Comment:
I don't think the two `Max` above are necessary since we can't batch calls anymore.

Reply:
Eh i had removed them but I had to fight with git, looks like this modification has gone.
Will delete it once more.